### PR TITLE
File upload status (v2 backport)

### PR DESCRIPTION
**Pull request by Boy132**
A backport of v2's file upload status. (https://github.com/pterodactyl/panel/commit/c6dccc568d91d95e6b5fa4dcaad4dfd532b6a5f6)
I only changed two things:
1. Changed the timeout before a completed upload is removed from 5 to 2 seconds. (_UploadButton.tsx#L111_)
2. Added the total and loaded bytes next to the name. (_FileManagerStatus.tsx#L61_)

file_upload

### CODE REVIEW

**Comment by KaspianDev** [comment]
After applying this patch drag & drop no longer works.

**Comment by Boy132** [comment]
> After applying this patch drag & drop no longer works.

I can't reproduce that with the given information. ("no longer works" isn't very descriptive)
You maybe got some conflicting third party mod or forget some changes when applying?

**Comment by KaspianDev** [comment]
> > After applying this patch drag & drop no longer works.
> 
> I can't reproduce that. You maybe got some conflicting third party mod or forget some changes when applying?

I run ptero panel from latest commit (as of this message), no mods

Edit: Tried again on 1.9.2, same result

**Comment by camwhi-te** [comment]
I'm running into the same issue with this (although my instance is modified).

**Comment by Boy132** [comment]
What browser are you using? What files are you using? (amount of files, size of files, file types etc.)
Also, can you try on an unmodified panel? Just to be sure.

**Comment by KaspianDev** [comment]
> What browser are you using? What files are you using? (amount of files, size of files, file types etc.) Also, can you try on an unmodified panel? Just to be sure.

Firefox, one 10KB yml file

Works fine on unmodified panel

**Comment by Jelcoo** [comment]
This seems to be a problem on the Firefox browser.
The first browser is Firefox, the second one is Microsoft Edge.
https://user-images.githubusercontent.com/53396500/177564610-b9d6fe76-294f-4b9c-af3f-b8cd5431d34a.mp4

File: resources/scripts/components/server/files/UploadButton.tsx [hunk 1]
```diff
@@ -19,18 +18,39 @@ const InnerContainer = styled.div`
...
     const directory = ServerContext.useStoreState((state) => state.files.directory);
+    const appendFileUpload = ServerContext.useStoreActions((actions) => actions.files.appendFileUpload);
+    const removeFileUpload = ServerContext.useStoreActions((actions) => actions.files.removeFileUpload);
 
     useEventListener(
         'dragenter',
         (e) => {
+            if (!isFileOrDirectory(e)) {
+                return;
+            }
```
Comment (Jelcoo) [inline resources/scripts/components/server/files/UploadButton.tsx:54]: It seems to be a problem with these 3 lines. When I remove them it works again.

**Comment by Boy132** [comment]
Turns out Firefox also adds a `application/x-moz-file` file to the drag event that caused these issues.

**Comment by KaspianDev** [comment]
I updated the patch, nothing changed.

Firefox 103.0b9 (64-bit)

Edit: uploading files from file manager is fine, but from the "downloads" popup in firefox no, (but that one still works in original pterodactyl, without patch). Maybe there's something else we forgot about?

Edit 2: This might be the fix, https://stackoverflow.com/a/10703964/13768631

**Comment by KaspianDev** [comment]
@DaneEveritt compat issues with that one still aren't fixed.

#4279

**Pull request merged**